typescript/infra/config/environments/mainnet3/chains.ts

```diff
@@ -34,23 +31,6 @@ export const chainMetadataOverrides: ChainMap<Partial<ChainMetadata>> = {
       gasPrice: 1 * 10 ** 9, // 1 gwei
     },
   },
-  ethereum: {
-    blocks: {
-      confirmations: 3,
-    },
-    transactionOverrides: {
-      maxFeePerGas: 150 * 10 ** 9, // gwei
-      maxPriorityFeePerGas: 5 * 10 ** 9, // gwei
-    },
-  },
-  scroll: {
-    transactionOverrides: {
-      // Scroll doesn't use EIP 1559 and the gas price that's returned is sometimes
-      // too low for the transaction to be included in a reasonable amount of time -
-      // this often leads to transaction underpriced issues.
-      gasPrice: 2 * 10 ** 9, // 2 gwei
-    },
-  },
```

Comment:
removing scroll gasPrice override entirely, will monitor the situation 

if this is unfruitful, we can add back at 0.2 gwei (which is still about 2-5x more than usual averages)